services/common/src/components/documents/spatial/AddSpatialDocumentsModal.tsx

```diff
@@ -95,18 +99,35 @@ const AddSpatialDocumentsModal: FC<AddSpatialDocumentsModalProps> = ({
             Please confirm that the composed map below correctly represents your selected spatial
             data files.
           </Typography.Paragraph>
-          <ViewSpatialDetail spatialDocuments={existingDocuments} />
+          {filesHaveGeomarkIds() && <ViewSpatialDetail spatialDocuments={existingDocuments} />}
         </>
       ),
     },
   ];
 
+  const addBundleIdToFiles = ({
+    docman_bundle_guid,
+    geomark_id,
+  }: {
+    docman_bundle_guid: string;
+    geomark_id: string;
+  }) => {
+    const newFiles = currentValues[fieldName].map((f) => {
+      return {
+        ...f,
+        docman_bundle_guid,
+        geomark_id,
+      };
+    });
+    dispatch(change(modalFormName, fieldName, [...initialDocuments, ...newFiles]));
```

Comment:
I remember that I had to take the `initialDocuments` _out_ of the dispatch-change-modalFormName. Otherwise I'd upload a spacial bundle, save, go to upload another, and my first bundle would already be there. Have you tested it when there's already a bundle there and you're adding another?

Reply:
I'm thinking that the geomark_id just wants to be a state variable or something. Then we're not having to add it to project summary documents and can keep it with the bundle.